extensions/mapstore/src/test/java/com/hazelcast/mapstore/GenericMapStoreTest.java

```diff
@@ -86,22 +89,30 @@ public void after() {
             mapStore.destroy();
             mapStore = null;
         }
+
+        try (SqlResult result = sqlService.execute("SHOW MAPPINGS")) {
+            for (SqlRow next : result) {
+                Object name = next.getObject(0);
+                logger.warning("Mapping " + name + " exists after test, dropping");
+                sqlService.execute("DROP MAPPING IF EXISTS " + name).close();
+            }
+        }
```

Comment:
I have added this, just to be sure.

I think there are still some possible issues where something fails and the `mapStore.destroy()` isn't called.

Reply:
The tests are executed one by one (`SerialTest`), the map store is being destroyed after each test with additional check, and finally `SimpleTestInClusterSupport#supportAfter` tries to remove any leftover. Defensive programming is just the best way of hiding bugs :)